velox/functions/sparksql/String.h

```diff
@@ -845,4 +845,76 @@ struct TranslateFunction {
   }
 };
 
+/// Decodes `string` in 'application/x-www-form-urlencoded' format using UTF-8
+/// encoding scheme.
+template <typename T>
+struct UrlDecoderFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
```

Comment:
What's the difference between this function and UrlDecodeFunction (velox/functions/prestosql/URLFunctions.h) Presto function?

Reply:
Hi @mbasmanova, I just noted this presto function. Looks we can reuse it for spark. Will close this PR soon. Thanks so much!